### ISSUE TITLE
Enhance error message

### ISSUE THREAD

**Issue by matthewpersico** [koalaman/shellcheck#1292]
#### For bugs
- Rule Id (if any, e.g. SC1000): SC2217
- My shellcheck version (`shellcheck --version` or 'online'): online
- [x] I tried on shellcheck.net and verified that this is still a problem on the latest commit
- [ ] It's not reproducible on shellcheck.net, but I think that's because it's an OS, configuration or encoding issue

#### Here's a snippet or screenshot that shows the problem:

```sh
#!/bin/bash

echo <<EOSSS
See http://osxdaily.com/2014/07/10/set-screen-saver-keyboard-shortcut-mac/
 if you need to rebuild the screen-saver-via-keystroke dohickey again.
 You can determine this by looking at Terminal/Services/Start Screen Saver
 to see if the Start Screen Saver service is there with keystroke
is CtrlCommand(Window)l

EOSSS
```

#### Here's what shellcheck currently says:
```
Line 3	SC2217: Redirecting to 'echo', a command that doesn't read stdin. Bad quoting or missing xargs?
```
#### Here's what I wanted or expected to see:
```
Line 3	SC2217: Redirecting to 'echo', a command that doesn't read stdin. Bad quoting, missing xargs or try using 'cat' instead?
```

**Comment by koalaman** [comment]
`echo` is probably the most common command this warning triggers for, so it's worth a special case. Thanks!
